.github/workflows/images.yml

```diff
@@ -0,0 +1,142 @@
+name: Images
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
I believe Docker Hub gave us builds for branches too, but I think this is reasonable to avoid chewing up all our build minutes on image building.

Reply:
Yep, my thoughts exactly. We never really made much use of the branch builds. But you reminded me to delete the docs about this.